cypress/integration/instance/InstanceEdit.ts

```diff
@@ -50,5 +50,65 @@ onlyOn(isEnvironmentType(EnvType.Mocked), () => {
             });
         });
     });
+
+    it("Switch from 'Webhook' to 'Ignore' Error handler", () => {
+      //Open 'Instance one' page
+      cy.ouiaId("Instances list table", "PF4/Table")
+        .ouiaId("Instance one", "PF4/TableRow")
+        .within(() => {
+          cy.get(
+            "td:first a[data-testid='tableInstances-linkInstance']"
+          ).click();
+        });
+
+      // Switch to error handling tab
+      cy.ouiaId("error-handling", "PF4/TabButton").click();
+
+      // Wait tab is loaded
+      cy.get(".pf-c-description-list__term").should("have.length", 5);
+      // Once tab is loaded start edit mode
+      cy.ouiaId("error-handling", "PF4/TabContent").within(() => {
+        cy.ouiaId("edit", "PF4/Button")
+          .should("have.attr", "aria-disabled")
+          .and("equal", "false");
+        cy.ouiaId("edit", "PF4/Button").click();
```

Comment:
@jomarko is possible that the edit button is disabled when the page is loaded? Do you expect DOM exception that you repeat `ouiaId("edit", "PF4/Button")`?

Reply:
yes the edit button is disabled when page is loaded. that is why I have this line in test
```
        cy.ouiaId("edit", "PF4/Button")
          .should("have.attr", "aria-disabled")
          .and("equal", "false");
```

That line above ensures I do not click on button soooner that in is enabled.

Click on disabled button was not throwing any error, but next lines of test were failing, because expected form fields/elements, were not present in the web page.

Reply:
thank you for your answer